project/src/pages/main/index.tsx

```diff
@@ -1,10 +1,13 @@
-import {Card} from '../../components/card/Card';
+import {CardProps} from '../../components/App/App';
+import {OffersList} from '../../components/OffersList/OffersList';
+import {FC} from 'react';
 
 type MainScreenProps = {
-  cardsCount: number
+  cardsCount: number,
+  offers: CardProps[]
 }
 
-export const MainScreen = ({cardsCount}: MainScreenProps): JSX.Element => (
+export const MainScreen:FC<MainScreenProps> = ({cardsCount, offers}) => (
```

Comment:
```suggestion
export const MainScreen = ({cardsCount, offers}:  MainScreenProps): JSX.Element => (
```
либо везде пиши FC либо так